README.md

```diff
@@ -56,23 +56,22 @@ Stuff that you're not allowed to bring on a plane or otherwise need to send via
 
 
 ### Items that are explicitly forbidden to bring on a plane
-- Ammunition
-- Fireworks
-- Explosives
+- Lighters (*only* allowed in hand luggage)
+- Matches
 - Lighter benzine
 - Lighter gas (Butane)
-- Matches
-- Lighters (*only* allowed in hand luggage)
 - Butane cartridges
 - Propane cartridges
+- Ammunition
+- Fireworks
+- Explosives
 - Radioactive medication
-- Toxics
+- Acids
 - Lead acid batteries
+- Toxics
 - Mercury (Quecksilber)
-- Acids
 - Bleach
 - Peroxides
-- Samsung Galaxy Note 7
```

Comment:
Why remove the “- Samsung Galaxy Note 7”? It is *actually* an explicitly named item forbidden to bring on board by airlines. (It's not mentioned here for plain entertainment, like some other things obviously are.)

Reply:
Do they still exist in the wild, though?